README.md

```diff
@@ -25,27 +25,53 @@ Add your API-Key from http://steamcommunity.com/dev/apikey
 
 Now in your file add the following at the top:
 
-    <?php
+```php
```

Comment:
It's not a typo, it's the way of telling GitHub wich sintax will you be using for code highlighting